src/app/helpers/storybook/ampDecorator/index.test.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import AmpDecorator from './index';
+
+describe('AmpDecorator', () => {
+  it('should render correctly', () => {
+    const component = AmpDecorator(() => <div>Foobar</div>);
+    expect(component.props.children).toMatchSnapshot();
```

Comment:
Can we snapshot the whole component, not just children? Also, there is a [snapshot utility that we should use](https://github.com/BBC-News/simorgh/blob/latest/src/app/helpers/tests/testHelpers.js#L5-L10)

Reply:
I wasn't able to use this the utility as Helmet stuff doesn't show, so you just get `<div><div> Foobar </div></div>`, I have have changed to snapshotting the whole component though